client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java

```diff
@@ -318,6 +321,11 @@ public void getConfig(String configId, String tenantId, ActionListener<MLConfig>
         client.execute(MLConfigGetAction.INSTANCE, mlConfigGetRequest, getMlGetConfigResponseActionListener(listener));
     }
 
+    public void createConversation(String name, ActionListener<CreateConversationResponse> listener) {
+        CreateConversationRequest createConversationRequest = new CreateConversationRequest(name);
```

Comment:
Test coverage is missing, I saw you added a unit test. Could you please check why it's not reflecting?